### ISSUE TITLE
Sticky spinner below viz table selection

### ISSUE THREAD

**Issue by trob** [joomlahenk/fabrik#483]
**To Reproduce**
Tested with Googlemap and Fullcalendar: After table selection it's still showing the spinner

**Additional context**
Nevertheless vizs are working, so it's only a layout issue.

**Comment by skurvish** [comment]
Yes, I have looked at this a couple of times. The spinner is being hidden when the table is chosen, but somebody is unhiding it and I have yet to determine who. I placed a mutation observer on it but the only mutations I see once the table is selected are style changes to hidden, not to display.

I will have another look at some point, but as you say it does not affect the behaviour.
